x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/alerts_grouping.tsx

```diff
@@ -189,7 +173,15 @@ const GroupedAlertsTableComponent: React.FC<AlertsTableComponentProps> = (props)
     [dispatch, props.tableId]
   );
 
-  const fields = useMemo(() => Object.values(sourcererDataView.fields || {}), [sourcererDataView]);
+  const indexNames = useMemo(() => props.dataViewSpec.title, [props.dataViewSpec]);
+  const runtimeMappings = useMemo(
+    () => props.dataViewSpec.runtimeFieldMap as RunTimeMappings,
+    [props.dataViewSpec]
+  );
```

Comment:
I'm not sure why we need to memoize these. IMO, creating functions on every render will be more resource intensive than than just accessing the spec fields. Or are there side effects on `.runtimeFieldMap` and `.title` (e.g. getters)?

Reply:
good point, I removed those 2. I guess I had blindly done that to be consistent with the other lines below...
Thanks!